integration/tests/close_channel.test.ts

```diff
@@ -67,7 +73,36 @@ describe('Close channel', () => {
 
         expect(postFundBalance.rawBalance).toBe(preFundBalance.rawBalance - depositAmount);
 
-        const msg = await createCloseChannelMessage(identity.messageSKSigner, params.channelId, identity.walletAddress);
+        let msg = await createResizeChannelMessage(identity.messageSKSigner, {
+            channel_id: params.channelId,
+            allocate_amount: depositAmount,
+            resize_amount: -depositAmount,
+            funds_destination: identity.walletAddress,
+        });
+
+        const resizeResponse = await ws.sendAndWaitForResponse(msg, getResizeChannelPredicate(), 1000);
+        const { params: resizeResponseParams } = parseResizeChannelResponse(resizeResponse);
+
+        expect(resizeResponseParams.state.allocations).toBeDefined();
+        expect(resizeResponseParams.state.allocations).toHaveLength(2);
+        expect(String(resizeResponseParams.state.allocations[0].destination)).toBe(identity.walletAddress);
+        expect(String(resizeResponseParams.state.allocations[0].amount)).toBe('0');
+        expect(String(resizeResponseParams.state.allocations[1].destination)).toBe(CONFIG.ADDRESSES.GUEST_ADDRESS);
+        expect(String(resizeResponseParams.state.allocations[1].amount)).toBe('0');
+
+        const {txHash: resizeChannelTxHash} = await client.resizeChannel({
+            ...composeResizeChannelParams(
+                resizeResponseParams.channelId as Hex,
+                resizeResponseParams,
+                state,
+            ),
+        });
+        expect(resizeChannelTxHash).toBeDefined();
```

Comment:
You can use `TestNitroClient.topUpChannel(...)` for that, but it obviously needs to be renamed and tweaked a bit.

Reply:
Do as you wish, I don't want to dive deeper there))0)